.travis.yml

```diff
@@ -1,6 +1,6 @@
 matrix:
     include:
-        - dist: trusty
+        # - dist: trusty
```

Comment:
Please just delete this line, it's not coming back , considering trusty is now EOL :-)